godtools/Localizable.strings

```diff
@@ -0,0 +1,24 @@
+/* 
+  Localizable.strings
```

Comment:
@frett and I just looked at this, and he shared one concern I think that's important enough to change this. We can run into trouble if we need to change or correct the phrase some day. If we had to change the key portion, then translators would have to translate the new phrase key, or we would have to do admin work in OneSky to update the key.

I really like the extension so we can just call `.localized`, but I think we need to use more traditional keys (`language_settings_title`)